drivers/volume/portworx/portworx.go

```diff
@@ -1778,6 +1781,287 @@ func (d *portworx) ValidatePureVolumesNoReplicaSets(volumeName string, params ma
 	return nil
 }
 
+type pureLocalPathEntry struct {
+	WWID        string
+	SinglePaths []string
+	Size        uint64
+}
+
+func GetSerialFromWWID(wwid string) (string, error) {
+	if !strings.Contains(wwid, "24a937") {
```

Comment:
nit: you might want was to extract `"24a937"` as a constant, since you keep using it on multiple places.

Just curious, I assume the WWID is coming from SCSI-inquiry, pages 0x80/0x83 ?  How do we "get it out" (ioctl()'s or external commands?)

Reply:
This WWID is actually produced from device-mapper multipath: it's `/dev/mapper/<WWID>` which contains the serial.

Good point, let me see if there's an existing constant